text/0000-inline-asm.md

```diff
@@ -952,17 +956,6 @@ fn mul(a: u64, b: u64) -> u128 {
 }
 ```
 
-## Use AT&T syntax on x86
-
-x86 is particular in that there are [two widely used dialects] for its assembly code: Intel syntax, which is the official syntax for x86 assembly, and AT&T syntax which is used by GCC (via GAS). There is no functional difference between those two dialects, they both support the same functionality but with a [different syntax][gas-syntax]. This RFC chooses to use Intel syntax since it is more widely used and users generally find it easier to read and write.
```

Comment:
It's probably worth keeping the note about Intel syntax being the official syntax, as that adds weight to our decision to use it by default.

The alternative, to use the platform's assembler's default syntax is appealing, until you realize that this would probably (IIUC) suggest using Intel syntax on `windows-msvc` to match MASM, and AT&T syntax on `*-gnu` to match GAS, which would definitely be the worst of both worlds.